server.go

```diff
@@ -141,7 +156,17 @@ func Respond(w http.ResponseWriter, r *http.Request, v interface{}) {
 	case render.ContentTypeJSON:
 		render.JSON(w, r, v)
 	default:
-		render.XML(w, r, v)
+		t, ok := r.Context().Value(TemplateKey).(*mustache.Template)
+		if ok {
+			html, err := t.Render(v)
+			if err != nil {
+				render.Render(w, r, ErrInternalServer(err))
+			} else {
+				render.HTML(w, r, html)
+			}
+		} else {
+			render.XML(w, r, v)
```

Comment:
I'm wondering if we wanna just stop doing XML render and just render a Not Found error here?

Reply:
Also maybe we wanna switch these around and render HTML when the `Accept` header is html and then default to JSON. If we are only going to do JSON and HTML right now that would make sense to me since you don't have to type extra stuff in curl and the browser will always send the correct headers.

Reply:
Yeah I like this much better. I just wanted to really apply the minimal set of changes here - so keep everything XML except for the 2 routes. However it'd be a smaller PR and more feature focussed if it was HTML then JSON.